nomnigraph - easy - use new test utils in converter_nomnigraph_test

duc0 opened the pull request:
Summary: Use new test utils in converter_nomnigraph_test , and add utils to set device option name, external inputs, outputs.

Differential Revision: D13586228

ZolotukhinM commented on caffe2/core/test_utils.h:
```diff
@@ -140,6 +144,9 @@ class NetMutator {
   }
 
+  // Set device name for the last created op.
+  NetMutator& setDeviceOptionName(const string& name);
```
Is `string` from `std`? It looks a bit inconsistent that in some places we have `std::string` and in others just `string`.

duc0 committed:
nomnigraph - easy - use new test utils in converter_nomnigraph_test (#15751)

Summary:
Use new test utils in converter_nomnigraph_test , and add utils to set device option name, external inputs, outputs.
Pull Request resolved: https://github.com/pytorch/pytorch/pull/15751

Differential Revision: D13586228

fbshipit-source-id: 0808cd0a75b5e108885c34dccddcdc348e4fa5bc
